TSOClient/tso.simantics/VMContext.cs

```diff
@@ -54,6 +54,21 @@ public VMArchitecture Architecture
         public VMAmbientSound Ambience;
         public ulong RandomSeed;
 
+        /// <summary>
+        /// A simple work around for TS1 moveobjects cheat
+        /// </summary>
+        public bool TS1_EnableMod //to clean up changes see VMNetDeleteObjectCmd Verify and VMEntity IsUserMovable
```

Comment:
Really don't like this - cheat state should be a child object to avoid cluttering VMContext, but I suppose we'll get there.